docs/docsite/rst/plugins/inventory.rst

```diff
@@ -31,6 +31,12 @@ If the plugin is in a collection, use the fully qualified name:
    [inventory]
    enable_plugins = namespace.collection_name.inventory_plugin_name
 
+If you want to be able to add a plugin from a collection to your other inventory methods, add it into the existing list:
+
+.. code-block:: ini
+
+   [inventory]
+   enable_plugins = host_list, script, auto, yaml, ini, toml, namespace.collection_name.inventory_plugin_name
```

Comment:
i would note that if plugin author uses the `plugin: fqcn` convention this is not needed as the `auto` plugin should just load the appropriate plugin

Reply:
OK, I'll note that, instead of the convoulted route I took, you should instead make sure you're using the fqcn.